### ISSUE TITLE
Compilation issue

### ISSUE THREAD

**Issue by lrozenblyum** [peholmst/I18N4Vaadin#13]
The project doesn't compile in Eclipse Mars.1 under JDK 1.0.60
The bundles look like not being generated.
Maybe using some specific maven-compiler-plugin settings are needed?

![screeni18n](https://cloud.githubusercontent.com/assets/1285898/10606492/fe60e15c-773b-11e5-8245-328ce3631cdd.png)

**Comment by peholmst** [comment]
Yes, the annotation processor is not invoked for some reason, or they are invoked but the generated output directories are not included as source folders in Eclipse. 

Check if 'target/generated-sources/annotations' contains anything and whether it is included as a source folder in Eclipse.

**Comment by lrozenblyum** [comment]
Unfortunately this directory is simply not created by Eclipse/Maven/m2e/whoever was responsible for its creation...
Even generated-sources doesn't exist in demo/target 

The only place where I could find anything at least particularly similar is :
cdi/tests-bundle-per-package/target/generated-sources (but without annotations subdir)
cdi/tests-bundle-per-class/target/generated-sources (but without annotations subdir)
